pkg/firewall/firewall_test.go

```diff
@@ -100,3 +103,50 @@ func TestCachesActiveKeepMembers(t *testing.T) {
 		)
 	}
 }
+
+func TestCachesInactiveKeepMembers(t *testing.T) {
```

Comment:
I am really confused why did I name the test `TestCachesActiveKeepMembers`...

Can we rename it to `TestCachesHasMinimumStake` and this test to `TestCachesHasNoMinimumStake`?